app/controllers/QuestionnaireController.scala

```diff
@@ -166,4 +166,9 @@ class QuestionnaireController(applicationClient: ApplicationClient, cacheClient:
       updateProgress()(_ => onSuccess)
     }
   }
+
+  private def universityMessageKey(implicit app: CachedDataWithApp) = app.application.applicationRoute match {
+    case ApplicationRoute.Edip => "currentUniversity"
+    case ApplicationRoute.Faststream => "university"
```

Comment:
The defaulting would be good in some ways, as "university" makes sense in most cases (_probably_ SDIP also). I am always a bit hesitant in defaulting when it would be more suitable for the developer to make an explicit decision, rather than accidentally omitting parameters and receiving a default that may or may not make sense